src/dashboard/apps/consent/utils.py

```diff
@@ -25,3 +27,21 @@ def consent_end_date(
     return datetime.datetime.now(datetime.timezone.utc).replace(
         month=12, day=31, hour=23, minute=59, second=59, microsecond=0
     )
+
+
+def duplicate_consent_end_date() -> datetime.datetime:
+    """Calculate the end date of the consent period for duplicated consents."""
+    duplicate_end_date = consent_end_date(CONSENT_NUMBER_DAYS_END_DATE)
+    if not CONSENT_NUMBER_DAYS_END_DATE:
+        duplicate_end_date = duplicate_end_date.replace(
+            year=duplicate_end_date.year + 1
+        )
```

Comment:
```suggestion
        duplicate_end_date += timedelta(year=1)
```

Reply:
`timedelta` does not support years :-/

I could use `relativedelta` from the `python-dateutil` package, but I don't like the idea of ​​installing an additional package just for this specific use case..

Reply:
ok then.